refinery/visualization_manager/views.py

```diff
@@ -535,16 +526,6 @@ def get_file_name(nodeuuid, samp_file=None, is_file_uuid=False):
                 FileStoreItem.MultipleObjectsReturned) as e:
             logger.error("Could not fetch FileStoreItem from %s: %s",
                          nodeuuid, e)
```

Comment:
With this change, have our expectations around `is_file_uuid` changed? As it was, either temp_fs would have been set, or there would have been an error, and that's no longer the case.

Reply:
This whole function (`create_igv_session`) appears to be used only by `igv_session` view which in turn appears to be unused (it looks like it was used for Java-based IGV).

Reply:
We definitely don't need `create_igv_session` anymore I'd remove it and any usages as well